### PR TITLE
fix(cwc): update telemetry

### PR DESCRIPTION
**Pull request by div5yesh**
- update telemetry as per new models
- send time between chunks

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/codewhispererChat/controllers/chat/telemetryHelper.ts [hunk 1]
```diff
@@ -359,21 +389,19 @@ export class CWCTelemetryHelper {
...
-    //         }
-
-    //         const trimmed = JSON.stringify(chunkDeltaTimes).slice(0, 2_000)
-    //         const fixed = trimmed.endsWith(',') ? trimmed.slice(0, -1) : trimmed
-    //         return fixed.endsWith(']') ? fixed : `${fixed}]`
-    //     } catch (e) {
-    //         return '[-1]'
-    //     }
-    // }
+    private getResponseStreamTimeBetweenChunks(tabID: string): number[] {
```
Comment (leigaol) [inline src/codewhispererChat/controllers/chat/telemetryHelper.ts:392]: Can you add doc string to explain why return [-1] at catch(e)? Also can you add log when catching an exception?

File: src/codewhispererChat/controllers/chat/telemetryHelper.ts [hunk 2]
```diff
@@ -291,7 +305,7 @@ export class CWCTelemetryHelper {
             cwsprChatReferencesCount: message.codeReferenceCount,
             cwsprChatFollowUpCount: message.followUpCount,
             cwsprChatTimeToFirstChunk: this.getResponseStreamTimeToFirstChunk(message.tabID),
-            cwsprChatTimeBetweenChunks: '', // this.getResponseStreamTimeBetweenChunks(message.tabID), //TODO: allow '[', ']' and ',' chars
+            cwsprChatTimeBetweenChunks: JSON.stringify(this.getResponseStreamTimeBetweenChunks(message.tabID)),
```
Comment (leigaol) [inline src/codewhispererChat/controllers/chat/telemetryHelper.ts:308]: Can this field be parsed correctly at service end?

Comment (div5yesh) [reply src/codewhispererChat/controllers/chat/telemetryHelper.ts:308]: Yes, it is an array of numbers. Required model changes have been made.

File: src/codewhispererChat/controllers/chat/telemetryHelper.ts (hunk 1 above)
Comment (div5yesh) [reply src/codewhispererChat/controllers/chat/telemetryHelper.ts:392]: Done. Added log. Returning empty array now. `[-1]` was an artifact of previous code changes.

**Review by hayemaxi — APPROVED** [review]

**Comment by justinmk3** [comment]
If this is still needed, please rebase and re-open this PR.

**Pull request closed without merging**